gandalf.js

```diff
@@ -6,33 +6,33 @@
  *
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!